app/validators/url_validator.rb

```diff
@@ -0,0 +1,16 @@
+class UrlValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return unless value.present?
+
+    record.errors[attribute] << (options[:message] || 'must be a valid URL') unless url_valid?(value)
+  end
+
+  def url_valid?(url)
+    url = begin
```

Comment:
I think `url = URI.parse(url) rescue false` would be enough here

Reply:
Rubocop changed that automagically.

Reply:
ok, let's not mess with RuboCop 🤖

Reply:
I already did. lol. I don't like that rule

Reply:
haha ok )